urnai/environments/stablebaselines3/custom_env.py

```diff
@@ -49,6 +49,8 @@ def reset(
         obs = self._env.reset()
         self._obs = obs
         obs = self._state.update(self._obs)
+        self._reward.reset()
+        self._reward.get(self._obs, 0, False, False)
```

Comment:
I added this `reward.get()` statement after `reset()` since the implementation of the collectable's reward class starts with a `previous_state` set to `None`. Without it, the first reward that an agent gets will always be 0. However, I'm not sure that all implementations of the `reward` class will have this behavior (of needing the `get` statement after resetting). Is there a way to improve this somehow? Or is it ok to leave the code as it is?